src/java/org/apache/cassandra/cql3/statements/schema/CreateIndexStatement.java

```diff
@@ -52,6 +52,18 @@
     private final IndexAttributes attrs;
     private final boolean ifNotExists;
 
+    private static final String DSE_INDEX_WARNING = "Index %s was not created. DSE custom index (%s) is not " +
+                                                    "supported. Consult the docs on alternatives (SAI indexes, " +
+                                                    "Secondary Indexes).";
+
+    private static final Set<String> DSE_INDEXES = ImmutableSet.of(
```

Comment:
Would it make sense to make this available to the test class?

Reply:
changed